vpt/processors/gaze_engagement_estimator/__init__.py

```diff
@@ -8,11 +8,13 @@
 
 class GazeEngagementEstimator(ProcessorBase[np.ndarray]):
     '''Transforms the gaze estimation to engagement estimation.'''
-    subj = Subject()
+    _subj: Subject
 
     def __init__(self, rotation_vector_source: ProcessorBase[np.ndarray]):
-        rotation_vector_source.get_data_stream().subscribe(lambda v:
-                                                             self.subj.on_next(np.linalg.norm(v)))
+        rotation_vector_source.get_data_stream().subscribe(
+            lambda v: self._subj.on_next(np.linalg.norm(v))
```

Comment:
Can you guarantee that this callback will never be called before the next line (initializing `self._subj`) runs? Or should they be swapped?